packager/media/formats/mp2t/mp2t_media_parser_unittest.cc

```diff
@@ -163,6 +172,22 @@ TEST_F(Mp2tMediaParserTest, TimestampWrapAround) {
   EXPECT_GT(video_max_dts_, static_cast<int64_t>(1) << 33);
 }
 
+TEST_F(Mp2tMediaParserTest, PtsZeroDtsWrapAround) {
+  // "bear-640x360.ts" has been transcoded from bear-640x360.mp4 by applying a
```

Comment:
It appears you quote the wrong file name.  This should be "bear-640x360_ptszero_dtswraparound.ts".